runtime/src/lib.rs

```diff
@@ -159,6 +159,7 @@ parameter_types! {
 		})
 		.avg_block_initialization(AVERAGE_ON_INITIALIZE_RATIO)
 		.build_or_panic();
+	pub const SS58Prefix: u8 = 66;
```

Comment:
Open https://github.com/crustio/crust/issues/506 for differ different chains

Reply:
Sure.